pydicom/config.py

```diff
@@ -60,6 +60,10 @@ def DS_decimal(use_Decimal_boolean=True):
 and datetime.time respectively. Default: False
 """
 
+empty_value = None
+"""Defines the value that an empty data element is set to
```

Comment:
Maybe we should start by defaulting to `''` to maintain current behaviour and put a comment in the release notes that the default behaviour will change in v1.5 or v1.6.

Reply:
Yes, I've thought about this, too. That would certainly be less disrupting.

Reply:
Ok, done.

Reply:
I have to re-think how to configure this, as besically each possible configuration has to be tested. So, maybe I will just support setting `''` or `None` to text VR values, and always set `None` to the rest. Maybe allow `b''` for binary values optionally, too.